modules/doc/content/newsletter/2020_05.md

```diff
@@ -1 +1,24 @@
 # MOOSE Newsletter (May 2020)
+
+## Stochastic Tools
+
+1. The surrogate system within the stochastic tools modules was enhanced to support the creation
+   of trainer and evaluation objects. This includes the ability to store and load arbitrary training
+   data for both in memory evaluation or loading from a file.
+1. The documentation for the stochastic tools module was improved to include more examples, including
+   using the new train/evaluate object structure, see [modules/stochastic_tools/index.md].
+
+## Python parameters package
+
+A new python package for input parameters was created in the python directory. This new package is
+thoroughly tested and will replace the various other parameter systems within the MOOSE python tools
+in the near future. Please consider using this new package if you are creating python tools in an
+application and desire a parameter system.
+
+## Finite Volumes
+
+Experimental support for solving PDEs using the finite volume method has been added to MOOSE.
```

Comment:
Is there any documentation yet?

Reply:
Yeah - but by pointing it out we increase the chance that somebody tries to use FV and thus increase the chances that our APIs get harder to change. It's better to have as few users as possible ;-)

Reply:
Haha and I love that you've put this at the bottom of newsletter...